lambda/lib/resize.ts

```diff
@@ -1,38 +1,37 @@
-import sharp from "sharp";
+/* eslint-disable @typescript-eslint/consistent-type-definitions */
```

Comment:
TODO: fix the rule

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/consistent-type-definitions.md#options